### ISSUE TITLE
Large line spacing in student's program code in teacher's "Inspect Submission" view

### ISSUE THREAD

**Issue by atilante** [apluslms/a-plus#936]
Teacher's "Inspect Submission" view in A+: if student's program code submission has more than 9 lines, the line numbers of the program code are wrapped onto two or more rows, resulting in double line height.

![program_code_submission_inspect](https://user-images.githubusercontent.com/3034945/145964586-68f24893-65ee-46a8-99ba-dceaee277dcc.jpg)

I encountered this with Firefox 95.0 on Ubuntu with 100% Zoom. If I decrease the browser's zoom to 90%, the problem disappears.

The CSS for the line numbers seems to have word wrap on, which might cause the problem.

In short: minor UI issue which should be easy to fix.

**Comment by Mankro** [comment]
Closing as duplicate: #794. You may comment there if you wish (but you don't have to since the issues are linked).
It is good to hear that the browser's zoom may be able to circumvent this. We weren't able to fix this because the bug does not occur on everyone's computers or browsers. But there must be something wrong in the CSS styles.
